src/main/java/domain/Ladder.java

```diff
@@ -0,0 +1,57 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Ladder {
+
+    private static final int UNIQUE_COUNT = 1;
+    private static final int FIRST_LINE_INDEX = 0;
+
+    private final List<Line> lines;
+
+    public Ladder(final List<Line> lines) {
+        validate(lines);
+        this.lines = new ArrayList<>(lines);
+    }
+
+    private void validate(final List<Line> lines) {
+        validateLineSizeEmpty(lines);
+        validateLinesSameSize(lines);
+    }
+
+    private void validateLineSizeEmpty(final List<Line> lines) {
+        if (lines.size() == 0) {
+            throw new IllegalArgumentException("사다리의 높이는 1 이상이어야 합니다");
```

Comment:
```suggestion
        if (lines.isEmpty()) {
            throw new IllegalArgumentException("사다리의 높이는 1 이상이어야 합니다");
```

이렇게 사용하면 매직넘버도 같이 해결할 수 있겠네요

Reply:
헉 생각도 못 하고 있었네요! 반영하겠습니다😊